python/dgl/network.py

```diff
@@ -153,3 +174,121 @@ def _recv_nodeflow(receiver, graph):
         return res
     else:
         return NodeFlow(graph, res)
+
+
+################################ Distributed KVStore Components ################################
+
+
+class KVMsgType(Enum):
+    """Type of kvstore message
+    """
+    FINAL = 1
+    INIT = 2
+    PUSH = 3
+    PULL = 4
+    PULL_BACK = 5
```

Comment:
what is pull back?

Reply:
PULL_BACK is the message sent from server when the client invoke a PULL message.